web/models.py

```
@@ -2790,3 +2790,34 @@ class ScheduledPost(models.Model):
+
+class VoiceChatParticipant(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    room = models.ForeignKey(VoiceChatRoom, on_delete=models.CASCADE)
+    joined_at = models.DateTimeField(auto_now_add=True)
+    is_speaking = models.BooleanField(default=False)
+    is_muted = models.BooleanField(default=False)
+
+    class Meta:
+        unique_together = ("user", "room")
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)